$MirOS: ports/audio/flac/patches/patch-doc_html_Makefile_in,v 1.1 2005/12/29 20:29:26 tg Exp $
--- doc/html/Makefile.am.orig	Tue Jan 25 05:35:42 2005
+++ doc/html/Makefile.am	Wed Feb  1 16:36:44 2006
@@ -55,4 +55,7 @@ uninstall-local:
 	rm -rf $(DESTDIR)$(docdir)/api
 maintainer-clean-local:
 	rm -rf api
+else
+install-data-local:
+	: # Nothing here so far...
 endif
